web/css/label.css

```diff
@@ -0,0 +1,34 @@
+:root {
+  --label-background-color: #37414c;
+  --label-box-shadow-heigh: -6px;
+  --label-translate-y: calc((-1 * var(--button-box-shadow-heigh)) - 2px);
+}
+
+.label {
+  height: 20px;
+  min-width: 60px;
+  padding: 8px 8px;
+  border-radius: 10px;
+  display: flex;
```

Comment:
display flex is have no use if the label remain a simple <p> ; text-align: center will do the job